docs/reference/src/documentation/language/built-ins/constants.md

```diff
@@ -0,0 +1,11 @@
+# Constants
+
+A constant in Sway is an `associated item` which is an item associated to a particular struct type.
```

Comment:
This sentence seems misleading right? Constants are more than just "associated constants for struct types"